src/LandingPage.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import Container from 'react-bootstrap/lib/Container';
+
+class LandingPage extends Component {
```

Comment:
Will this component have state? Why `class` for this vs. pure function for `Dashboard`?